### ISSUE TITLE
ISSUE: Starting with Visual Studio 2022, Visual Studio no longer includes .NET Framework components for .NET Framework 4.0 - 4.5.1 because these versions are no longer supported.

### ISSUE THREAD

**Issue by Protiguous** [AdamWhiteHat/BigDecimal#50]
**Description**
According to the page, [page](https://learn.microsoft.com/en-us/dotnet/framework/configure-apps/how-to-enable-and-disable-automatic-binding-redirection): Visual Studio 2022 and later versions can't build apps that target .NET Framework 4.0 through .NET Framework 4.5.1.

**Expected Behavior**
Full error-free and warning-free builds using Visual Studio 2022.

**Actual Behavior**
Full builds using Visual Studio 2022.

**Steps To Reproduce**
Clone the main branch.
Open the solution using Visual Studio 2022. (_I'm using Visual Studio 2022 Preview._)

**Additional context/Comments/Notes/Rants**
It seems we can still compile BigDecimal even when the versions NET45 and NET46 are in the project file.
So, I'm not sure if this issue actually applies?

Just a heads-up. Are you sure that we need this backwards compatibility?

**Comment by AdamWhiteHat** [comment]
Yeah, I don't know what its on about, as I run VS 2022 and, as you noticed, it seems to build .NET 4.5 just fine. In fact, just to prove the point, I changed the target versions and I can build assemblies for the .NET Framework 4.0 through 4.5 without issue.

Yet the infobox on the page you linked seems to state very clearly that it "cannot build apps that target .NET Framework 4.0 through .NET Framework 4.5.1". Odd.|

Perhaps they mean from installing 2022 from its installer. I know that after I installed VS 2022, I downloaded the .NET 4.0 and 4.5 frameworks with SDKs, and I presume you'd need to do that to compile applications targeting such .NET versions.

Ill try and see if I can get some statistics/numbers surrounding how many users are regularly targeting what .NET versions, but I suspect there is still a decent number of people using these unsupported versions.

Since I can still build them, and I'm the one who's publish the NuGet packages anyways, Id prefer to keep making them available to support people targeting old framework versions.
